### ISSUE TITLE
Level: "Cliff Hanger"

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#142]
```
:name=Cliff Hanger
:description=
:num_rabbits=6
:num_to_save=6
:block=2
:climb=1
:dig=4
:bridge=2
:bash=2
:explode=2
:rabbit_delay=10
:music=tryad-witness
###########
#        O#
#         #
#         #
#         #
#        Q#
# *       #
###     ###
###########
:*=ccccc
```
Easy one to introduce a concept.

**Comment by andybalaam** [comment]
The concept of climbing a wall and getting into an exit that is hanging?  Are we happy with that?

**Comment by tttppp** [comment]
I'm happy with it. Longer term the falling past exit and climbing into exit behaviours could be made more memorable with animation.

**Comment by andybalaam** [comment]
Yes, I like this.

**Comment by andybalaam** [comment]
Nice level. Maybe reduce the number of rabbits? My solution is `bridge;(8,6);climb;(7,3)` was that the intended one?

**Comment by andybalaam** [comment]
Note that solution was hand typed on my phone so is subject to errors.

**Comment by colonelfazackerley** [comment]
see my solution here
https://github.com/colonelfazackerley/rabbit-escape/blob/arcade2/rabbit-escape-engine/src/rabbitescape/levels/arcade/02_Cliff-hanger.rel

yours looks a little different

**Comment by colonelfazackerley** [comment]
but that's fine.

It's not supposed to be hard.

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a
